Lib/test/test_concurrent_futures.py

```diff
@@ -568,6 +568,19 @@ def test_shutdown_no_wait(self):
         # shutdown.
         assert all([r == abs(v) for r, v in zip(res, range(-5, 5))])
 
+    def test_hang_issue45021(self):
+        """https://bugs.python.org/issue45021"""
```

Comment:
Please explain the purpose of the test with a short sentence. You can refer to the bug using `bpo-45021: (...)` syntax.